site/docs/v1/tech/apis/reactor.adoc

```diff
@@ -0,0 +1,122 @@
+---
+layout: doc
+title: Reactor APIs
+description: APIs for activating, deactivating, and retrieving the status of a license.
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+This page contains the APIs for Reactor. Here are the APIs:
+
+* <<Activate Reactor license>>
+* <<Retrieve Reactor status>>
+* <<Regenerate Reactor license>>
+* <<Deactivate Reactor license>>
+
+
+== Activate Reactor license
+
+This API is used to activate a Reactor license.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Activate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/reactor#
+--
+
+==== Request Body
+
+[.api]
+[field]#licenseId# [type]#[String]# [required]#Required#::
+The license Id to to activate.
+
+[field]#license# [type]#[String]# [optional]#Optional#::
+The Base64 encoded license value.
+
+=== Response
+
+This API does not return a JSON response body.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:never_search_error!:
+
+== Retrieve Reactor status
+
+This API is used to retrieve the status of Reactor features. This also includes breached password metrics.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve Reactor status
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/reactor#
+--
+
+=== Response
+
+:no_errors:
+:never_search_error:
+include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
+:no_errors!:
+:never_search_error!:
+
+include::docs/v1/tech/apis/_reactor-response-body.adoc[]
+
+
+== Regenerate Reactor license
+
+This API is used to regenerate the Reactor license.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Regenerate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#PUT# [uri]#/api/reactor#
+--
+
+=== Response
+
+This API does not return a JSON response body.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
+:never_search_error!:
+
+
+== Deactivate Reactor license
+
+This API is used to deactivate a Reactor license.
```

Comment:
What are the implications of doing this to my runner server?

Reply:
Correct, all licensed features will be un-available. If you were using breach password detection, this will no longer work, you will not be able to select licensed features, and existing configurations may no longer work if they are configured to use licensed features. 

In practice, there probably aren't many reasons to do this, the one use case may if you re-generated the license from another instance of FusionAuth and you are using the same license key here, you would have to de-commission and then activate again with a new key.